src/operation/problem_results.jl

```diff
@@ -411,32 +454,135 @@ function read_realized_aux_variables(res::ProblemResults)
     return _read_realized_results(res.aux_variable_values, nothing)
 end
 
-function read_realized_variables(res::ProblemResults, variable_keys)
-    return _read_realized_results(
-        res.variable_values,
-        [VariableKey(x...) for x in variable_keys],
+"""
+Return the values for the requested variable keys for a problem.
+Accepts a vector of keys for the return of the values. If the time stamps and keys are
+loaded using the [load_results!](@ref) function it will read from memory.
+
+# Arguments
+- `variables::Vector{<:OptimizationContainerKey}` : keys of desired results
```

Comment:
Docstring needs updating - don't refer to keys.

Reply:
Fixed via ed3223440a445d69e20ccb4f915c70c15bbeb82f